lib/ansible/modules/packaging/os/yum.py

```diff
@@ -343,15 +342,15 @@ def is_group_env_installed(name):
     return False
 
 
-def is_installed(module, repoq, pkgspec, conf_file, qf=None, en_repos=None, dis_repos=None, is_pkg=False, installroot='/'):
+def is_installed(module, pkgspec, conf_file, rpm=False, qf=None, en_repos=None, dis_repos=None, is_pkg=False, installroot='/'):
```

Comment:
I don't think it's safe to change when we use the yum api, however I don't remember the specifics.  I would not change this behaviour just to change this as I think it exists for some cornercases.